The Many Suits of Command

juxjux9930 opened the pull request:
🆑
rscadd: New COS, CMO, CE CSO, XO and CO hardsuit sprites. 
rscdelete: Removes unused pathfinder hardsuit obj sprites.
/🆑

I've done all the suits. Open to sprite commentary.

https://github.com/Baystation12/Baystation12/pull/29475/checks?check_run_id=1190714927

juxtaposed9930 commented:
I lied. I'm touching the CE suit, only slightly though.
Well, maybe a bit more than slightly.

juxtaposed9930 commented:
I've done it. For better or worse, I've done it.

No more cookie-cutter command HCMs.

juxtaposed9930 commented:
Small tweaks to sprites that I _thought_ I pushed but didn't.

juxtaposed9930 commented:
Right, I'm done making my own little nitpicky changes.

juxtaposed9930 commented:
Tweaked colors in the CO/XO onmob in response to feedback

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -306,12 +390,12 @@
+		bomb = ARMOR_BOMB_MINOR,
+		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_SHIELDED
 		)
```
```suggestion
	)
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -297,6 +315,72 @@
+		/obj/item/rig_module/vision/meson,
+		/obj/item/rig_module/grenade_launcher/mfoam,
+		/obj/item/rig_module/cooling_unit
+		)
```
```suggestion
	)
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -297,6 +315,72 @@
+	glove_type = /obj/item/clothing/gloves/rig/command/engineering
+	boot_type = /obj/item/clothing/shoes/magboots/rig/command/engineering
+
+	allowed = list(/obj/item/weapon/gun,
+				   /obj/item/device/flashlight,
+			   	   /obj/item/weapon/tank,
+				   /obj/item/device/suit_cooling_unit,
+				   /obj/item/weapon/storage/toolbox,
+				   /obj/item/weapon/storage/briefcase/inflatable,
+				   /obj/item/weapon/inflatable_dispenser,
+				   /obj/item/device/t_scanner,
+				   /obj/item/weapon/rcd)
```
```suggestion
	allowed = list(
		/obj/item/weapon/gun,
		/obj/item/device/flashlight,
		/obj/item/weapon/tank,
		/obj/item/device/suit_cooling_unit,
		/obj/item/weapon/storage/toolbox,
		/obj/item/weapon/storage/briefcase/inflatable,
		/obj/item/weapon/inflatable_dispenser,
		/obj/item/device/t_scanner,
		/obj/item/weapon/rcd
	)
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -297,6 +315,72 @@
+	suit_type = "engineering command hardsuit"
+	desc = "A high-tech hardsuit rig control module issued to ranking officers of the Expeditionary Corps and their peers."
+	icon_state = "command_eng_rig"
+	armor = list(
+		melee = ARMOR_MELEE_RESISTANT,
+		bullet = ARMOR_BALLISTIC_SMALL,
+		laser = ARMOR_LASER_SMALL,
+		energy = ARMOR_ENERGY_SMALL,
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_SHIELDED,
+		rad = ARMOR_RAD_SHIELDED
+		)
```
```suggestion
	armor = list(
		melee = ARMOR_MELEE_RESISTANT,
		bullet = ARMOR_BALLISTIC_SMALL,
		laser = ARMOR_LASER_SMALL,
		energy = ARMOR_ENERGY_SMALL,
		bomb = ARMOR_BOMB_PADDED,
		bio = ARMOR_BIO_SHIELDED,
		rad = ARMOR_RAD_SHIELDED
	)
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -243,20 +257,22 @@
+		bomb = ARMOR_BOMB_RESISTANT,
+		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_SHIELDED
 		)
```
```suggestion
	)
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -197,20 +203,22 @@
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_SMALL
 		)
```
```suggestion
	)
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -114,15 +114,15 @@
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_SMALL
 		)
```
```suggestion
	)
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -74,15 +74,15 @@
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_SMALL
 		)
```
```suggestion
	)
```

Spookerton commented on maps/torch/items/rigs.dm:
```diff
@@ -12,12 +12,12 @@
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_SMALL
 		)
```
```suggestion
	)
```

Spookerton reviewed (dismissed):
code style since we're here

juxjux9930 committed:
New Command HCMs & bye bye duplicates

Newer CE suit because I don't like my lame original


Resprites the CMO hardsuit


Slight tweak to CSO sprite & fixes OBJs having 4 directions

I couldn't stop laughing at the sheer girth of the lad.
New CO & XO hardsuit body sprites.


Creates & implements a new CE suit, keeping old intact


Apply suggestions from code review

Co-authored-by: Spookerton <spkrtn@pm.me>

juxtaposed9930 commented:
Squashed

juxtaposed9930 commented:
Right, should be the _last_ last adjustment according to feedback - mostly just making some things on sprites clearer visually

juxtaposed9930 commented:
I said that, but I lied. I touched the kneepads of the CO and XO suits again.

juxtaposed9930 commented:
I know not everyone is a fan of this alternative COS suit, hence the separate commit - if enough people say no to it, I'll revert to the previous, more mil-themed suit.

Personally, I feel this "sporty" suit fits the EC theme more - they're a uniformed service, but they're still very flashy with their uniforms and public image. This suit is definitely more in-you-face and elaborate, and less utilitarian.

juxtaposed9930 commented:
I've also removed gitkraken from my task bar, so unless I go out of my way to find the exe there shouldn't be any more tiny modifications nobody asked for.

juxtaposed9930 commented:
Reverted back to the old commit.

juxtaposed9930 commented:
You know, this should've been clear to me sooner - making all these changes means I'm not entirely happy, and if I'm not, no way others will be. Taking down until they're _ready_.
